Templates/Per Tenant Extension/.github/workflows/CreateOnlineDevelopmentEnvironment.yaml

```diff
@@ -38,13 +39,25 @@ env:
 
 jobs:
   Initialization:
+    needs: [ ]
     runs-on: [ windows-latest ]
     outputs:
       deviceCode: ${{ steps.authenticate.outputs.deviceCode }}
       telemetryScopeJson: ${{ steps.init.outputs.telemetryScopeJson }}
       githubRunner: ${{ steps.ReadSettings.outputs.GitHubRunnerJson }}
       githubRunnerShell: ${{ steps.ReadSettings.outputs.GitHubRunnerShell }}
     steps:
+      - name: Input
+        if: github.event_name == 'workflow_dispatch'
+        run: |
+          Write-Host "Inputs:"
+          $eventPath = Get-Content -Encoding UTF8 -Path $env:GITHUB_EVENT_PATH -Raw | ConvertFrom-Json
+          $eventPath.inputs.psObject.Properties | Sort-Object { $_.Name } | ForEach-Object {
+            $property = $_.Name
+            $value = $eventPath.inputs."$property"
+            Write-Host "- $property = '$value'"
```

Comment:
Is this needed?

Reply:
This is for support purposes - when people are running a workflow, it is hard to find out what input they typed - they also do not remember themselves the exact input and finding it embedded in other output is hard.
This is why I added this allover (could be a mini-action as suggested by Maria)